x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/types.ts

```diff
@@ -31,3 +31,22 @@ export interface SystemCellComponentMap {
 }
 
 export type SystemCellId = keyof SystemCellComponentMap;
+
+type CaseHooks = (props?: Record<string, unknown>) => {
+  open: ({ attachments }: { attachments: any[] }) => void;
+  close: () => void;
+};
+
+export interface CasesService {
```

Comment:
When a [case package](https://github.com/elastic/kibana/issues/151648) is created these types will be imported from the package.